packages/fiori/src/ProductSwitch.js

```diff
@@ -7,6 +7,7 @@ import ProductSwitchTemplate from "./generated/templates/ProductSwitchTemplate.l
 
 // Styles
 import ProductSwitchCss from "./generated/themes/ProductSwitch.css.js";
+import "./DefaultTheme.js";
```

Comment:
we should name this DefaultThemeParameters (or something else, but in essence it is about parameters)

also to avoid mistakes by component developers who forget to include it, we could move this import to the generated css.js file (line above).